src/Tizen.Multimedia.MediaPlayer/Player/PlayerEnums.cs

```diff
@@ -132,6 +132,8 @@ public enum PlayerState
         /// <summary>
         /// Preparation in progress.
         /// </summary>
+        /// <remarks>The Preparing state is translated to <see cref="PlayerState.Ready"/> internally.
+        /// And in this state, other methods and properties cannot be set.</remarks>
```

Comment:
How about below? Because ```this``` in second line made me confused `Preparing` with `Ready`.

```
/// <remarks>
/// In this state, other methods and properties cannot be set.
/// And it's translated to <see cref="PlayerState.Ready"/> internally.
/// </remarks>
```